### PR TITLE
Added the merde command using the same joke as git stash poop.

### PR DESCRIPTION
**Pull request by JoeyKanaly**
#12

### CODE REVIEW

File: shellGitTypo.sh [hunk 1]
```diff
@@ -151,6 +151,16 @@ git() {
...
+    
+  elif [ "$1" = "merde" ]; then
+    echo "
+  )  )
+ (   ) (
+  ) _   )
+   ( \_
+ _(_\ \)__
+(____\___))"
+    command git merge "${@:2}" "${@:3}"
```
Comment (Xaxetrov) [inline shellGitTypo.sh]: I don't understand why you write both `"${@:2}"` and `"${@:3}"`. Can you explain this ?

**Review by Xaxetrov — COMMENTED** [review]
Thanks for your PR !

File: shellGitTypo.sh (hunk 1 above)
Comment (JoeyKanaly) [reply shellGitTypo.sh]: That's my bad I misunderstood how the parameter passing was working, I removed it now.

Comment (Xaxetrov) [reply shellGitTypo.sh]: OK nice!

**Pull request merged**